src/averageAges.js

```diff
@@ -37,7 +41,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const arrWoman = (!withChildren)
```

Comment:
Because in array more than one woman.
``suggestion
  const womenArray = (!withChildren)
```